webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala

```diff
@@ -2578,18 +2637,14 @@ case class FileValueWithSipiMetadata(fileValue: FileValueV2, sipiFileMetadata: F
  */
 object FileValueWithSipiMetadata {
   def fromJsonLdObject(
```

Comment:
Definitely rename, maybe completely remove?

This is not getting the json ld object anymore. Not it is merely combining two constructors.